include/megaapi_impl.h

```diff
@@ -882,6 +884,7 @@ class MegaPushNotificationSettingsPrivate : public MegaPushNotificationSettings
 
     void enableChat(MegaHandle chatid, bool enable) override;
     void setChatDnd(MegaHandle chatid, int64_t timestamp) override;
+    void setChatsDnd(int64_t timestamp) override;
```

Comment:
Shouldn't it be `setGlobalChatsDnd()` to be in line with the getter and checker (`isGlo...()`) methods?